### ISSUE TITLE
Extract option should allow relativepath for `--serviceControlUrl`

### ISSUE THREAD

**Issue by ramonsmits** [Particular/ServicePulse#303]
If the user already know that is wants to setup a reverse proxy by using Application Request Routing then the user wants to enter a relative path:

    ServicePulse.Host.exe --extract --serviceControlUrl="/api" --outPath="C:\inetpub\websites\ServicePulse\1.4.1"

This is not possible, you will get the following error:

> The value specified for 'serviceControlUrl' is not a valid URL

The workaround it to not specify the argument and manually edit `app.constants.js`.

**Comment by robbihun** [comment]
Agreed; additionally, if you're following the guidance on hosting behind IIS from [this page](https://docs.particular.net/servicepulse/install-servicepulse-in-iis) specifically, the upgrade option does not work without having to manually edit the app.constants.js as you cannot use the --servicecontrolurl or --servicecontrolmonitoringurl switches.

![image](https://user-images.githubusercontent.com/138997/63458265-1dd0e780-c418-11e9-9e54-31b8c71cfa93.png)

**Comment by WilliamBZA** [comment]
We have looked at this and will be working to get it included in one of the upcoming maintenance releases for this package. It's definitely an area that can do with some improvement
